denizen_scripts/survival/repo-link/MobScripts/SpiderStuff.dsc

```diff
@@ -0,0 +1,10 @@
+Paranoia_poison_canceller:
+  type: world
+  debug: false
+  events:
+    on player damaged by cave_spider:
+      - if <context.damager.is_mythicmob> && <context.damager.entity> == cave_spider:
```

Comment:
if `on player damaged` is by `cave_spider`, then `<context.damager.entity>` will always equal `cave_spider`, we don't have to check it within the check here
```suggestion
      - if <context.damager.is_mythicmob>:
```